### PR TITLE
Query stats doc

### PR DESCRIPTION
**Pull request by Dfinity-skaestle**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [x] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [x] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/smart-contracts/advanced-features/query-stats.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+When users interact with a dapp, query calls may be used. Query calls cannot modify a canister's state, making it impossible to track them from the canister's code. 
+
+The **query stats** feature gives developers information about the use of each canister's query call endpoints. With this feature, an approximation of some statistics related to query stats are made available to developers as part of the existing canister status API.
+
+:::caution
+Since query statistics collected in the canister status are approximations, they might not capture all query calls, especially for infrequently used canisters. Query stats are incremented periodically by the system, and therefore lag behind the actual query execution by up to one hour.
+:::
+
+Query stats currently do not include cached query calls.
```
Comment (Sawchord) [inline docs/developer-docs/smart-contracts/advanced-features/query-stats.md:13]: No longer true, after current change

File: docs/developer-docs/smart-contracts/advanced-features/query-stats.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Query stats
+
+## Overview
+
+When users interact with a dapp, query calls may be used. Query calls cannot modify a canister's state, making it impossible to track them from the canister's code. 
+
+The **query stats** feature gives developers information about the use of each canister's query call endpoints. With this feature, an approximation of some statistics related to query stats are made available to developers as part of the existing canister status API.
```
Comment (Sawchord) [inline docs/developer-docs/smart-contracts/advanced-features/query-stats.md:7]: This makes it sound to me like we keep statistics per Endpoint, but we actually collect one set of stats per canister

**Comment by sa-github-api** [comment]
Dear @Dfinity-skaestle,

In order to potentially merge your code in this open-source repository and therefore proceed with your contribution, we need to have your approval on DFINITY's [CLA](https://github.com/dfinity/cla/blob/main/CLA.md).

If you decide to agree with it, please visit this [issue](https://github.com/dfinity/cla/issues/454) and read the instructions there. Once you have signed it, re-trigger the workflow on this PR to see if your code can be merged.

— The DFINITY Foundation

File: docs/developer-docs/smart-contracts/advanced-features/query-stats.md (hunk 2 above)
Comment (Sawchord) [reply docs/developer-docs/smart-contracts/advanced-features/query-stats.md]: Changed it

File: docs/developer-docs/smart-contracts/advanced-features/query-stats.md (hunk 1 above)
Comment (Sawchord) [reply docs/developer-docs/smart-contracts/advanced-features/query-stats.md]: Removed

**Comment by jessiemongeon1** [comment]
CLA check fails due to @Dfinity-skaestle account no longer being part of DFINITY group. Bypassing branch protection to merge, since this PR was created when [Stefan Kaestle](https://github.com/Dfinity-skaestle) was still at DFINITY.

**Pull request merged**